src/components/TextAreaInput/TextAreaInput.js

```diff
@@ -111,11 +105,11 @@ function PrivateTextAreaInput({
   return (
     <>
       <InputLabel name={name} labelCopy={labelCopy} />
-      <textarea
+      <BaseTextAreaInput
```

Comment:
FYI @ethantran I know that partnerships uses this component. I didn't make any functional changes, but wanted to tag you in case something weird happens.

I did try to clean up the props. I added some props related to the form engine were being used, but weren't listed as props for this component. Similarly, I removed props that weren't being used (e.g. `onBlur` was being defined within the component). Let me know if there's anything I need to change!

Reply:
Fwiw, I think in order to confirm this, @ethantran will have to update the EDS tag version to this one and spot check (as of now monorepo won't see these changes).